src/citrine/informatics/predictor_evaluation_metrics.py

```diff
@@ -0,0 +1,138 @@
+    """
+
+    @classmethod
+    def get_type(cls, data) -> Type[Serializable]:
+        """Return the subtype."""
+        t = data["type"]
+        return {
+            "RMSE": RMSE,
+            "NDME": NDME,
+            "StandardRMSE": StandardRMSE,
+            "PVA": PVA,
+            "F1": F1,
+            "AreaUnderROC": AreaUnderROC,
+            "CoverageProbability": CoverageProbability,
+        }[t]
+
+
+class RMSE(Serializable["RMSE"], PredictorEvaluationMetric):
+    """[ALPHA] Root-mean-square error
+
+    """
+    typ = properties.String("type", default="RMSE", deserializable=False)
+
+    def __repr__(self):
+        return "rmse"
+
+    def __str__(self):
+        return "RMSE"
+
+
+class NDME(Serializable["NDME"], PredictorEvaluationMetric):
+    """[ALPHA] Non-dimensional model error
+
+    """
+    typ = properties.String("type", default="NDME", deserializable=False)
+
+    def __repr__(self):
+        return "ndme"
+
+    def __str__(self):
+        return "NDME"
+
+
+class StandardRMSE(Serializable["StandardRMSE"], PredictorEvaluationMetric):
+    """[ALPHA] Standardized root-mean-square error
+
+    """
+    typ = properties.String("type", default="StandardRMSE", deserializable=False)
+
+    def __repr__(self):
+        return "standardized_rmse"
+
+    def __str__(self):
+        return "Standardized RMSE"
+
+
+class PVA(Serializable["PVA"], PredictorEvaluationMetric):
+    """[ALPHA] Predicted vs. actual data.
+    Results are returned as a flattened list, where each item represents
+    predicted vs. actual data for a single point.
+
+    """
+    typ = properties.String("type", default="PVA", deserializable=False)
+
+    def __repr__(self):
+        return "predicted_vs_actual"
+
+    def __str__(self):
+        return "Predicted vs Actual"
+
+
+class F1(Serializable["F1"], PredictorEvaluationMetric):
+    """[ALPHA] Support-weighted F1 score.
+
+    """
+    typ = properties.String("type", default="F1", deserializable=False)
+
+    def __repr__(self):
+        return "f1"
+
+    def __str__(self):
+        return "F1 Score"
+
+
+class AreaUnderROC(Serializable["AreaUnderROC"], PredictorEvaluationMetric):
+    """[ALPHA] Area under the receiver operating characteristic (ROC) curve.
+
+    """
+    typ = properties.String("type", default="AreaUnderROC", deserializable=False)
+
+    def __repr__(self):
+        return "area_under_roc"
+
+    def __str__(self):
+        return "Area Under the ROC"
+
+
+class CoverageProbability(Serializable["CoverageProbability"], PredictorEvaluationMetric):
+    """[ALPHA] Fraction of observations for which the magnitude of the error is within a
+    confidence interval of a given coverage level.
+
+    Parameters
+    ----------
+    coverage_level: Union[str, float]
+        Confidence-interval coverage level.
+        The coverage level must both be between 0 and 1.0 (non-inclusive) and specified
+        to 3 significant figures.
```

Comment:
Doesn't it only need to be specified to 3 sig figs if it is passed as a string?

Reply:
Previously, yes. I just pushed a batch of changes that allows the user to specify the string with < 3 significant figures. If they specify > 3 significant figures (as a float or a string) we will round to 3 decimal places and issue a warning.